media/js/app/taxonomy/taxonomy.js

```diff
@@ -75,6 +75,12 @@
                 var term = new Term({display_name: termName});
                 this.get('term_set').add(term);
             }
+        },
+        addTermWithURI: function(termName, uri) {
```

Comment:
The base addTerm function at line 73 is no longer referenced. Rather than add this new function, a simple refactor on the original addTerm would be more efficient.